src/type-translator.ts

```diff
@@ -157,6 +160,8 @@ export class TypeTranslator {
   public symbolToString(sym: ts.Symbol): string {
     // This follows getSingleLineStringWriter in the TypeScript compiler.
     let str = '';
+    let alias = this.symbolsToAliasedNames.get(sym);
+    if (alias) return alias;
```

Comment:
I can't think of it right now, nor how to even find it.  :(  I will find it myself at some point.